docs/articles/baseplatform/running-qemu-tests.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+.. _running-qemu-tests:
+
+Running tests on QEMU
+=====================
+
```

Comment:
Probably adding some description on where the tests are coming from is good to have.. maybe a brief elaboration of the testimage and the tests on local.conf